src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs

```diff
@@ -79,6 +81,17 @@ internal InfiniteScrollList(Lazy<JoinableTaskFactory> joinableTaskFactory)
 
             DataContext = Items;
             CheckBoxesEnabled = false;
+
+            _loadingStatusIndicator.PropertyChanged += LoadingStatusIndicator_PropertyChanged;
+        }
+
+        private void LoadingStatusIndicator_PropertyChanged(object sender, PropertyChangedEventArgs e)
+        {
+            if (e.PropertyName == nameof(LoadingStatusIndicator.Status))
+            {
+                ThreadHelper.ThrowIfNotOnUIThread();
+                LtbLoading.Text = _loadingStatusIndicator.LocalizedStatus;
```

Comment:
Not sure... but should we check if the value actually changed since this might fire NotifyPropertyChanged unnecessarily sometimes? It may be guaranteed not to happen by design of the indicator.

Reply:
Added if to change Text only if it is different.